addons/lunch/tests/test_ui.py

```diff
@@ -0,0 +1,11 @@
+
+
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import HttpCase, tagged
+
+@tagged('-at_install', 'post_install')
+class TestUi(HttpCase):
+    def test_01_ui(self):
+        self.start_tour("/", 'order_lunch_tour', login='admin', timeout=180)
```

Comment:
I think the tour will fail if it runs on a week-end and there are no products available.

Could you create at least one vendor + one lunch product that's available everyday before the call to `start_tour()` to prevent it from happening? :-)

Thanks!